### DISCUSSION TITLE
Entity is unknown error in log files

### DISCUSSION THREAD

**Discussion by siegeld** [Koenkk/zigbee2mqtt#21735] [General]
I have zigbee2mqtt running integrated with Home Assistant. I'm running zigbee2mqtt as an add on. It's working well! It uses mqtt to link Home Assistant and uses the topic prefix homeassistant. I have other systems that publish stuff to mqtt with the topic prefix homeassistant. What I don't understand is why zigbee2mqtt is printing out these entity is unknown error messages. It should just be ignoring them I would think as the messages have nothing to do with zigbee2mqtt. Any ideas?

**Comment by siegeld** [comment]
For example, here is a logged error message:

2024-03-09 14:48:13Entity 'homeassistant/resi/xyz' is unknown

**Reply by francisp2** [reply]
The homeassistant prefix is used for discovery messages, so use another prefix for normal operations, e.g zigbee2mqtt (which is the default)

**Reply by dimatx** [reply]
@francisp2 - Can you elaborate? I think my issue and the @siegeld's issue is that zigbee2mqtt is printing error messages about topics that are unrelated to any of the devices in zigbee2mqtt. The messages that generate the errors are not coming from zigbee2mqtt.

**Comment by dimatx** [comment]
I have a similar issue where when I use MQTT to publish to a button, it logs an error in zigbee2mqtt. The button comes from a HASS.agent running on a PC. It should have nothing to do with zigbee2mqtt.

`2024-04-15 10:43:39Entity 'homeassistant/button/Surface-0' is unknown`

This line is logged each time anything is published to that topic on MQTT.
